app/assets/stylesheets/alchemy/base.scss

```diff
@@ -142,31 +142,31 @@ kbd {
 }
 
 .top-rounded-border {
-  @include border-top-radius($default-border-radius);
+  border-top-radius: $default-border-radius;
```

Comment:
we should use [the Bourbon mixins](https://github.com/thoughtbot/bourbon/blob/v4-stable/app/assets/stylesheets/addons/_border-radius.scss) here, because they have a short hand for adding both `border-top-left-radius` and `border-top-right-radius` what we actually need here. Currently all "one sided" border radii went away from the GUI:

![alchemy cms - modal missing rounded borders](https://cloud.githubusercontent.com/assets/42868/16379270/d5eaabb8-3c70-11e6-8a63-17e660fe6030.jpg)

![alchemy cms - missing rounded borders](https://cloud.githubusercontent.com/assets/42868/16379273/db60df54-3c70-11e6-8c37-450145f5eb9b.jpg)